### PR TITLE
[tw] Add content/zh-tw/api-gateway.md

### LINKED ISSUE #2164: [tw] `API Gateway` for Traditional Chinese
+ Localize `API Gateway` in Traditional Chinese
+ https://glossary.cncf.io/api-gateway/

### PR DESCRIPTION
**Pull request by gamerslouis**
### Describe your changes
Origin content from https://glossary.cncf.io/api-gateway/

### Related issue number or link (ex: `resolves #issue-number`)
resolve #2164 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/api-gateway.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: API 閘道器
+status: Completed
+category: 技術
+tags: ["網路", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+[API](/application-programming-interface/)閘道器是一個工具，它聚集了多應用程式的API，使它們可以透過單一的入口存取。這使得組織能夠集中處理特定的功能（如身分驗證、授權或限制應用程式之間的請求數量）。API閘道器作為的公共介面，來為API使用者（通常是外部的）提供服務。
```
Comment (hwchiu) [inline content/zh-tw/api-gateway.md:10]: 這邊應該要導向中文版本的連結，可參閱其他 PR

File: content/zh-tw/api-gateway.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+如果您希望為外部使用者提供API，您通常會希望有一個單一的入口點來管理所有的存取。此外，API閘道器可以讓您統一套用功能到所有的流量上，而不需要修改任何應用程式的程式碼。
```
Comment (hwchiu) [inline content/zh-tw/api-gateway.md:14]: 英文跟中文之間補上空白，與其他 PR 維持一致性

File: content/zh-tw/api-gateway.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## 如何幫助 {#how-it-helps}
+
+API閘道器為應用程式中的各種API提供了一個單一存取點，這使得組織能夠在集中處理跨域商業邏輯或安全邏輯，也讓應用程式使用者能夠透過訪問單一地址滿足所有需求。API閘道器可以透過為系統中所有Web服務的請求提供單一的存取點來簡化像安全性和[可觀測性](/observability/)這樣的運營問題。由於所有請求都流經API閘道器，它可以集中添加功能，如指標收集、速率限制和授權。
```
Comment (hwchiu) [inline content/zh-tw/api-gateway.md:18]: 連結導向中文版本
Web也許可以翻譯成網頁

Comment (gamerslouis) [reply content/zh-tw/api-gateway.md:18]: 我覺得Web翻譯成網頁好像怪怪的，特別是在這邊通常是API、json之類的格式，翻譯成網頁 (web page) 感覺概念上有點落差？

**Review by pichuang — APPROVED** [review]
lgtm

**Review by hwchiu — APPROVED** [review]
LGTM

**Review by ydFu — APPROVED** [review]
LGTM

**Pull request merged**